internal/install/execution/platform_link_generator_test.go

```diff
@@ -45,6 +45,53 @@ func TestGenerateRedirectURL_InstallSuccess(t *testing.T) {
 	require.Equal(t, expectedURL, result)
 }
 
+func TestGenerateLoggingURL_InstallSuccess(t *testing.T) {
+	t.Parallel()
+
+	g := NewPlatformLinkGenerator()
+
+	// We set an API key in the unit test so we don't make an real HTTP request
+	// to the New Relic short URL service (see integration test), and so we can test
+	// the query param being added for the fallback installation strategy below.
+	g.apiKey = ""
+
+	infraEntityGUID := "MXxBUE18QVBQTElDQVRJT058OTE2NzQxNg"
+	infraRecipe := types.OpenInstallationRecipe{
+		Name:        "infrastructure-agent-installer",
+		DisplayName: "Infrastructure Agent",
+	}
+	logsRecipe := types.OpenInstallationRecipe{
+		Name:        "logs-integration",
+		DisplayName: "Logs integration",
+	}
+	agentInstalledStatus := &RecipeStatus{
+		DisplayName: "Infrastructure Agent",
+		Name:        "infrastructure-agent-installer",
+		Status:      RecipeStatusTypes.INSTALLED,
+		EntityGUID:  infraEntityGUID,
+	}
+	logsInstalledStatus := &RecipeStatus{
+		DisplayName: "Logs integration",
+		Name:        "logs-integration",
+		Status:      RecipeStatusTypes.INSTALLED,
+	}
+	installStatus := InstallStatus{
+		recipesSelected: []types.OpenInstallationRecipe{infraRecipe, logsRecipe},
+		Installed:       []*RecipeStatus{agentInstalledStatus, logsInstalledStatus},
+		EntityGUIDs:     []string{infraEntityGUID},
+		Statuses:        []*RecipeStatus{agentInstalledStatus, logsInstalledStatus},
+	}
+
+	launcherEncodedParams := "eyJxdWVyeSI6IlwiZW50aXR5Lmd1aWQuSU5GUkFcIjpcIk1YeEJVRTE4UVZCUVRFbERRVlJKVDA1OE9URTJOelF4TmdcIiJ9"
+	expectedRedirectURL := fmt.Sprintf("https://%s/redirect/entity/%s", nrPlatformHostname(), infraEntityGUID)
+	expectedLoggingLink := fmt.Sprintf("https://%s/launcher/logger.log-launcher?platform[accountId]=0&launcher=%s", nrPlatformHostname(), launcherEncodedParams)
```

Comment:
🤔 Solid test, however, I think we need the `[accountId]=0` to be dynamic to match the behavior in the tested function ([reference](https://github.com/newrelic/newrelic-cli/pull/1186/files#diff-f8343147dd77d4baf043d9670284714fdad6657bb697f28765d9400b5536e104R125)). This test fails if I run it locally because I have the `NEW_RELIC_ACCOUNT_ID` environment variable set...which makes wonder how this passed in CI 😆

Another option would be to narrow the scope of assertion to only check that `loggingLinkResult` contains the `launcherEncodedParams` (or some other substring/combo).